OsStub/BaseCryptLibMbedTls/Hash/CryptSha2.c

```diff
@@ -0,0 +1,403 @@
+/** @file
+  SHA-2 Digest Wrapper Implementations over OpenSSL.
+
+  This file includes SHA256, SHA384, SHA512, all part of SHA-2 family.
+
+Copyright (c) 2009 - 2016, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include "InternalCryptLib.h"
+#include <mbedtls/sha256.h>
+#include <mbedtls/sha512.h>
+
+enum InternalSha2Type{_SHA2_256, _SHA2_384, _SHA2_512};
+
+/**
+
+  Computes the size of a mbedtls_sha{256, 512}_context struct given a
+  Sha2Type. Note that 384 and 512 both use mbedtls_sha512_context.
+
+  @param[in]   type     Algorithm requested. Must be supported.
+
+  @retval UINTN > 0   The size of the corresponding context struct.
+  @retval UINTN == 0   Error; algorithm not supported.
+
+**/
+static UINTN GetContextSize(enum InternalSha2Type type)

```

Comment:
This coding style is not acceptable.

Please use:

STATIC
UINTN
GetContextSize (
  IN INTERNAL_SHA2_TYPE Type
  )